src/test/regress/expected/multi_function_in_join_0.out

```diff
@@ -173,6 +167,7 @@ CREATE TYPE min_and_max AS (
   maximum INT
 );
 SET client_min_messages TO DEBUG1;
+SET citus.enable_metadata_sync TO OFF;
```

Comment:
Thinking on this again,  I guess if it only changes the DEBUG messages, we should try to remove enable_metadata_sync = off OR add a comment why we disabled it. I think the former 

(We all agree about the tech-debt in the regression tests, I think it deserves extra few hours to help clean these. Otherwise, for future readers, it is very cryptic why we have these on/offs )